src/factory.js

```diff
@@ -222,6 +222,7 @@ const createFSPIOPErrorFromJoiError = (error, cause, replyTo) => {
       case 'date.format':
       case 'number.integer':
       case 'any.allowOnly':
+      case 'any.only':
```

Comment:
Please add a comment as to why this was necessary, perhaps include a link to the bug?